### ISSUE TITLE
Attunement Alter Probably Broken

### ISSUE THREAD

**Issue by yustjason1** [HellFirePvP/AstralSorcery#1588]
There is a slight problem when I'm trying to attune anything to Constellation: Mineralis. It seems as if one of the relay locations is directly on top of the actual attunement alter. Is there anyway to circumvent this? If not is there a command I can use to give myself a pre-attuned celestial crystal?

**Comment by yustjason1** [comment]
I guess the alter decided to work on the next night. Nevermind then...

**Comment by Danekjovax** [comment]
I am having this problem now.  Was working for several days (attuned 4 crystals to it previously), but now it's not working.  If it persists, I'll open a new ticket.

**Comment by PocketfulOfGeese** [comment]
As the astral tome lists, the altar works as a relay itself.

If you have issues @Danekjovax come to the discord for usage assistance. It's not a bug.

**Comment by Danekjovax** [comment]
> As the astral tome lists, the altar works as a relay itself.
> 
> If you have issues @Danekjovax come to the discord for usage assistance. It's not a bug.

Thanks for the offer.  I happened to observe exactly as you described, and looking more closely at my setup, I did see that one of the relays was not utilized (see attached).  Looking at the information on Mineralis I realized the nights it was not working was the dim nights (about 5 in a row), and finally it worked again.

Thanks for the feedback.  Love this mod, and plan to enjoy it going forward.  Peace! <>
![2022-04-16_21 45 22](https://user-images.githubusercontent.com/10191173/163888968-64dbf8dd-e8e2-4295-8e64-c3e0942da797.png)
<
